infra/aws/terraform/management-account/delegated-admins.tf

```
@@ -0,0 +1,52 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# https://docs.aws.amazon.com/organizations/latest/userguide/orgs_integrate_services_list.html
+
+resource "aws_organizations_delegated_administrator" "config_multiaccount" {
+  account_id        = module.security_audit.account_id
```

How about:
```suggestion
  account_id        = module.security_audit.account.id
```
and then don't change the `outputs.tf`

?

---

We can't technically do that with HCL since `module.security_audit` is an object and there is no `account` ojbect in the module. the `id` is private to the module so it is necessary to modify the module to expose it.

---

Insufficient coffee. Sure, that sounds right.